src/main/java/core/basesyntax/operation/ReturnOperation.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.operation;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class ReturnOperation implements OperationHandler {
+    @Override
+    public void processOperation(FruitTransaction fruitTransaction) {
+        int fruitsInStorage = Storage.storage.get(fruitTransaction.getFruit());
```

Comment:
Be careful with direct access to the storage map. If the fruit is not present in the map, this will cause a NullPointerException. You should check if the fruit is present in the storage before retrieving its quantity.